pkg/cmd/bazci/bazci.go

```diff
@@ -185,6 +185,7 @@ func processBuildEventProtocolLog(action, bepLoc string) error {
 			outputDir = filepath.Join("bazel-testlogs", outputDir)
 			summary := event.GetTestSummary()
 			for _, testResult := range testResults[label] {
+				outputDir := outputDir
```

Comment:
This looks like `outputDir` is just overwriting itself. What am I missing about how it works?

Reply:
`outputDir := outputDir` declares a new variable with the same name as the old one. It shadows the original variable in a new scope. That means we can update the value in the loop but each iteration will get the original value of `outputDir`.